doc/nrf/app_dev/config_and_build/sysbuild/sysbuild_forced_options.rst

```diff
@@ -65,10 +65,6 @@ Trying to change them directly in an image will result in the new value being ov
 +-------------------------------------------------------------------------+                                                                           |                         |
 | :kconfig:option:`CONFIG_CHIP_OTA_IMAGE_BUILD`                           +                                                                           |                         |
 +-------------------------------------------------------------------------+---------------------------------------------------------------------------+                         |
-| :kconfig:option:`CONFIG_SUIT_MPI_APP_RECOVERY`                          +               ``SB_CONFIG_SUIT_BUILD_RECOVERY``                           |                         |
-+-------------------------------------------------------------------------+                                                                           |                         |
-| :kconfig:option:`CONFIG_SUIT_MPI_RAD_RECOVERY`                          +                                                                           |                         |
-+-------------------------------------------------------------------------+---------------------------------------------------------------------------+-------------------------+
```

Comment:
This change results in the merge of the two rows in the last column. You can see the result here -> https://ncsdoc.z6.web.core.windows.net/PR-19016/nrf/app_dev/config_and_build/sysbuild/sysbuild_forced_options.html

For comparison, this is the current output of this file before your change -> https://docs.nordicsemi.com/bundle/ncs-latest/page/nrf/app_dev/config_and_build/sysbuild/sysbuild_forced_options.html

Is this change intended?

Reply:
No, thank you for spotting this!
Fixed.